meetings/santiago2019/index.md

```diff
@@ -114,18 +114,23 @@ _Note: This schedule is subject to change! Check here for updates._
 
 <tt>14:00 - EOD</tt> **Collaboration time**
 
-<tt>Evening</tt> **Trends and challenges in multiobjective optimization**
+<tt>18:00</tt> **Trends and challenges in multiobjective optimization**
 
 The Pontificia Universidad Católica de Valparaíso will hold a seminar on
-multiobjective optimization in Santiago. Attendees of JuMP-dev are invited.
-More details to come.
+multiobjective optimization in Santiago. This seminar will be held at the CEA (Centro de Estudios Avanzados) building in [Antonio Bellet #314, Providencia, Santiago](https://goo.gl/maps/gqa2g9fHVCq), and all attendees of JuMP-dev are invited.
 
 ### Friday
 
-<tt>Morning</tt> **Introductory tutorial**
+<tt>14:00</tt> **Introductory tutorial in Valparaíso**
 
 An introductory tutorial will be held in Valparaíso for those unable to attend
-Tuesday's introductory tutorial in Santiago. More details to come.
+Tuesday's introductory tutorial in Santiago. The tutorial will be held at the [Escuela de Ingeniería Informática](http://www.inf.ucv.cl) of [Pontificia Universidad Católica de Valparaíso](https://www.pucv.cl) ([Av Brasil 2241, Valparaiso, Chile](https://goo.gl/maps/VPHBJsP1qr62)) on Friday, March 15th at 14:00.
+
+### Monday, March 11
+
+<tt>15:00 - 13:30</tt> **JuMP@USM MISTI Workshop**
+
+An additional workshop focusing on the MISTI project sponsoring JuMP-Dev will be held the Monday just before the workshop. This workshop will be held at the Campus Santiago San Joaquín of Universidad Técnica Federico Santa María. For more information and registration form see the [JuMP@USM MISTI Workshop page](http://www.industrias.usm.cl/quienes-somos/operations-management/seminars/march-11-2019/).
```

Comment:
JuMP-Dev should be JuMP-dev for consistency

Reply:
fixed